docs/monoDrive_home/Simulator-Commands.md

```diff
@@ -0,0 +1,365 @@
+# Simulator API Commands
+
+## Get Simulator Version
+Retrieves the current version of the simulator and of the simulator API with the commands documented on this page.
+
+**command ID**:  "GetVersion"
+
+**command data**:
+```json
+{	
+}
+```
+**response**: (string)
+
+`simulator_version: 1.12, api_version: 4.0`
+
+
+## Simulator Configuration
+Used to configure a simulator session. This command includes the settings associated with the simulator session following the execution of the command and applies to all future sessions until the command is run again. The "pys_materials" settings affect the returns from lidar and radar sensors for the list of known materials in the scene.
+
+**command ID**: “SimulatorConfig_ID”
+
+**command data**:
+
+```json
+{
+	"id": (string) - used to manage configurations on the client side,
```

Comment:
The format of these is very different from the JSON given for each sensor. I would suggest providing examples of each commands input and then describing each parameter below similar to how the sensor configuration are done. e.g.

```json
{
    id: 0,
    data_thing_1: 1.0,
    data_thing_2: 2.0,
    ...
}
```

* **id(int)**: The Id of the the thing
* **data_thing_1(float)**: The data thing 1
* **data_thing_1(float)**: The data thing 1

Reply:
I think this is a great idea, @pedro- do you think it would be possible to get examples of the json configurations?